FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -376,110 +547,280 @@ paths:
                 $ref: '#/components/schemas/Response'
-        403:
-          description: Forbidden
+        404:
+          description: UUID not found
           content: 
             application/json:
               schema:
                 $ref: '#/components/schemas/Response'
         406:
           description: Not acceptable - only responds in application/json format
-          content:
+          content: 
             application/json:
               schema:
                 $ref: '#/components/schemas/Response'
         500:
           description: Server error
-          content:
+          content: 
             application/json:
               schema:
                 $ref: '#/components/schemas/Response'
       security:
-      - FaultRead:
-        - fraihmwork.fault.read
+      - SystemWrite:
+        - fraihmwork.system.write
 
-  /fault/{uuid}:
-    get:
+  /component/{uuid}/refresh:
+    post:
       tags:
-      - Fault
-      summary: Request existing fault info for specific fault
-      description: This endpoint allows requesting fault info by UUID
-      operationId: getFaultRegistration
+      - Components
+      - Component Updates
+      summary: Refresh existing component properties
+      description: Permits an already registered component to be refreshed without changing any properties
+      operationId: refreshComponentInformation
       parameters:
         - in: path
           name: uuid
           required: true
           schema:
             type: string
             format: uuid
-          description: Uuid of the faults being updated
+          description: Uuid of the component property being refreshed
       responses:
         200:
           description: Successful operation
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/FaultRegistrationInformation'
+                $ref: '#/components/schemas/Response'
         401:
           description: Not authenticated
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        404:
+          description: UUID not found
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        500:
+          description: Server error
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+      security:
+      - SystemWrite:
+        - fraihmwork.system.write
+
+  /fault:
+    post:
+      tags:
+      - Fault Updates
+      - Faults
+      summary: Posts a new fault that is affecting a system
+      description: Allows a system to report one or more faults currently affecting the system. Using this endpoint means that the client is not managing their own fault IDs, and the fault will require a new tracking UUID (that will be created and assigned in the response).
```

Comment:
"Allows a **component** to report one or more faults currently affecting the **component**. Using this endpoint means that the client is not managing their own fault IDs, and the fault will require a new tracking UUID (that will be created and assigned in the response)."

Reply:
Fixed